### DISCUSSION TITLE
使用ImageUploader组件  点击后会出现一个关于aria-hidden属性的报错

### DISCUSSION THREAD

**Discussion by gaoxiaozhu** [ant-design/ant-design-mobile#6684] [General]
引入组件后点击上传,会出现一个报错 但是不影响功能使用
Blocked aria-hidden on a <input> element because the element that just received focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden. <input accept=​"image/​*" type=​"file" class=​"adm-image-uploader-input" aria-hidden=​"true">​

查了一下这个属性  然后再源码中找到后注释就好了,不清楚这样做对不对

**Comment by gaoxiaozhu** [comment]
反正目前报错是没有了

**Comment by zombieJ** [comment]
upload 内部用 input 做文件处理，感觉这个本来就看不到的确是不需要这个 `aria-hidden`
